rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -423,15 +441,39 @@ s32 cellGemForceRGB(u32 gem_num, float r, float g, float b)
 	return CELL_OK;
 }
 
-s32 cellGemGetAccelerometerPositionInDevice()
+s32 cellGemGetAccelerometerPositionInDevice(u32 gem_num, vm::ptr<f32[4]> pos)
 {
-	UNIMPLEMENTED_FUNC(cellGem);
+	cellGem.todo("cellGemGetAccelerometerPositionInDevice(gem_num=%d, pos=*0x%x)", gem_num, pos);
+	const auto gem = fxm::get<gem_t>();
+
+	if (!gem)
+	{
+		return CELL_GEM_ERROR_UNINITIALIZED;
+	}
+
+	if (!check_gem_num(gem_num))
+	{
+		return CELL_GEM_ERROR_INVALID_PARAMETER;
+	}
+
 	return CELL_OK;
 }
 
 s32 cellGemGetAllTrackableHues(vm::ptr<u8> hues)
 {
 	cellGem.todo("cellGemGetAllTrackableHues(hues=*0x%x)");
+	const auto gem = fxm::get<gem_t>();
+
+	if (!gem)
+	{
+		return CELL_GEM_ERROR_UNINITIALIZED;
+	}
+
+	for (size_t i = 0; i < 360; i++)
+	{
+		hues[i] = 0;
```

Comment:
We're stubbing so why not `hues[i] = true;`?

Reply:
yes. maybe it should